druid/src/app_delegate.rs

```diff
@@ -43,6 +47,46 @@ impl<'a> DelegateCtx<'a> {
         let target = target.into().unwrap_or(Target::Global);
         self.command_queue.push_back((target, command))
     }
+
+    /// Create a new window.
+    /// `T` must represent the application state provided to [`AppLauncher::launch`].
+    ///
+    /// [`AppLauncher::launch`]: struct.AppLauncher.html#method.launch
+    pub fn new_window<T: Any>(&mut self, desc: WindowDesc<T>) {
+        if self.state_types.check_window_desc::<T>() {
```

Comment:
since we know this is a `WIndowDesc` and we're only worried about `T`, can we just store the typeid of `T` and compare that?